src/jaegertracing/ConfigTest.cpp

```diff
@@ -94,6 +94,17 @@ TEST(Config, testZeroSamplingParam)
     }
 }
 
+TEST(Config, testPropagationFormat)
+{
+    {
+        constexpr auto kConfigYAML = R"cfg(
+propagation_format: w3c
```

Comment:
Is the format of the config file documented somewhere?

Reply:
No, because I don't know where. Should I put it in examples/config.yml.

Reply:
I don't know either. Looks like it's not documented in the README, would be good to have a reference to examples/, and then include all properties in that sample config (comment them out if necessary). But this could be done via separate PR.